### PR TITLE
Ukman documentation

### PR DESCRIPTION
**Pull request by Naktakala**

### CODE REVIEW

File: ChiDoc/PAGES/ProgrammersManual/ReferenceContent/devman_A3_UnknownMan.h [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+- \ref devmanA3_sec3
+- \ref devmanA3_sec4
+
+
+\section devmanA3_sec0 Introduction
+As a developer you might come across code such as
+\code
+imap[i] = sdm.MapDOF(cell, i);
+\endcode
+or it's more elaborate cousin
```
Comment (ragusa) [inline ChiDoc/PAGES/ProgrammersManual/ReferenceContent/devman_A3_UnknownMan.h]: its

**Review by ragusa — APPROVED** [review]
Very useful. Thank you. There's a tiny typo that I noted but you can ignore if you want.

**Comment by ragusa** [comment]
@Naktakala I fixed it online. Can I merge this now?

**Comment by Naktakala** [comment]
> @Naktakala I fixed it online. Can I merge this now?

Yes!

**Pull request merged**